test/test_torch.py

```diff
@@ -3276,6 +3276,18 @@ def warn_fn():
             self.assertEqual(frameinfo.lineno - 6, warning.lineno)
             self.assertEqual(len(w), 1)
 
+    def test_warnOnce_caught(self, device):
```

Comment:
Add the `@onlyCPU` decorator since this test doesn't do anything with the device

Reply:
done